### DISCUSSION TITLE
Can't add secondary email in Github email setting.

### DISCUSSION THREAD

**Discussion by Soorajalipanhwar** [community/community#71288] [GitHub Education]
### Select Topic Area

General

### Body

I want to add a secondary email address on GitHub but it says it's already added although there is no secondary email in the setting. I want to get student benefits through GitHub student developer pack and I have a student email of my university but I am unable to add it in GitHub. Someone please guide.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Soorajalipanhwar! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
